server/protocols/jmap-rfc-8621/doc/README.md

```diff
@@ -0,0 +1,11 @@
+Annotated JMAP documentation
+============================
+
+This directory contains annotated JMAP documentation as found on https://jmap.io/.
+
+Officially finalized specifications so far regarding JMAP are:
+
+* [The core protocol](https://jmap.io/spec-core.html) [[RFC 8620](https://tools.ietf.org/html/rfc8620)]
+* [JMAP Mail](https://jmap.io/spec-mail.html) [[RFC 8621](https://tools.ietf.org/html/rfc8621)]
+
+Annotations aim at tracking implementation progress in James project.
```

Comment:
As a JMAP specification developer I need an indication of "how the annotation system works".